### PR TITLE
Update OG docs

### PR DESCRIPTION
**Pull request by wensi-ai**

### CODE REVIEW

File: docs/getting_started/building_blocks.md [hunk 1]
```diff
@@ -130,6 +112,24 @@ This demo lets you choose a scene from the BEHAVIOR dataset, loads it along with
     --8<-- "src/examples/scenes/scene_selector.py"
     ```
 
+### **Scene Tour Demo**
+!!! abstract "This demo is useful for..."
+
+    * Understanding how to load a scene into **`OmniGibson`**
```
Comment (cremebrule) [inline docs/getting_started/building_blocks.md:118]: update docstring

File: docs/getting_started/building_blocks.md [hunk 2]
```diff
@@ -130,6 +112,24 @@ This demo lets you choose a scene from the BEHAVIOR dataset, loads it along with
...
+    * Accessing all BEHAVIOR dataset scenes
+
+```{.python .annotate}
+python -m omnigibson.examples.scenes.scene_selector
+```
+
+This demo lets you choose a scene from the BEHAVIOR dataset, and move around the scene with keyboard control.
```
Comment (cremebrule) [inline docs/getting_started/building_blocks.md:125]: here too

File: docs/getting_started/building_blocks.md [hunk 3]
```diff
@@ -392,6 +343,44 @@ This demo loads in a single object, and then dynamically modifies its state so t
...
+
+    ``` py linenums="1"
+    --8<-- "src/examples/object_states/onfire_demo.py"
+    ```
+
+### **Overlaid Demo**
+!!! abstract "This demo is useful for..."
+
+    * Understanding how cloth objects can be overlaid on rigid objects
+    * Understanding current heuristics used for gauging a cloth's "foldness"
```
Comment (cremebrule) [inline docs/getting_started/building_blocks.md:368]: I don't think we measure folded in this demo

File: docs/getting_started/installation.md [hunk 4]
```diff
@@ -82,41 +84,72 @@ Then, simply launch the desired script:
...
 
         ```shell
         git clone https://github.com/StanfordVL/OmniGibson.git
         cd OmniGibson
         ```
 
-    4. Run the command to setup a virtual conda environment to run **`OmniGibson`**:
+    5. Setup a virtual conda environment to run **`OmniGibson`**:
 
+        On Linux (bash):
```
Comment (cremebrule) [inline docs/getting_started/installation.md:114]: try to use the tab feature (see HEADLESS / GUI above)

File: docs/getting_started/installation.md [hunk 5]
```diff
@@ -82,41 +84,72 @@ Then, simply launch the desired script:
...
+        ```shell
+        set OMNIGIBSON_NO_OMNIVERSE=1&&python omnigibson/scripts/setup.py&&set OMNIGIBSON_NO_OMNIVERSE=
+        ```
+
+    8. 🎉 Congrats! You installed **`OmniGibson`** successfully.  
 
 1. Be sure keep track of where you choose Omniverse to write package files! By default this should be `~/.local/share/ov/pkg`
 
-2. If you installed Isaac Sim to the default location, this is `~/.local/share/ov/pkg/isaac_sim-2022.1.1`
+2. Cache and Nucleus is not required for our project. It is sufficient to install Isaac Sim after installing Omniverse Launcher
```
Comment (cremebrule) [inline docs/getting_started/installation.md:149]: I think you can delete this -- Isaac Sim auto installs Cache IIRC, though if it works without cache / nucleus with no noticeable side effects, maybe it's okay? @ChengshuLi do you have cache / nucleus installed on your machine?

File: omnigibson/examples/objects/highlight_objects.py [hunk 6]
```diff
@@ -1,15 +1,12 @@
...
 import omnigibson as og
 
 
 def main(random_selection=False, headless=False, short_exec=False):
     """
-    Highlights visually all object instances of some given category and then removes the highlighting
+    Highlights visually all object instances of windows and then removes the highlighting
     It also demonstrates how to apply an action on all instances of objects of a given category
-    ONLY WORKS WITH OPTIMIZED RENDERING (not on Mac)
+    ONLY WORKS WITH OPTIMIZED RENDERING
```
Comment (cremebrule) [inline omnigibson/examples/objects/highlight_objects.py:9]: i think we can remove this line

**Review by cremebrule — APPROVED** [review]
LGTM! after you fix the minor comments, feel free to merge :D

File: docs/getting_started/building_blocks.md (hunk 1 above)
Comment (wensi-ai) [reply docs/getting_started/building_blocks.md:118]: done

File: docs/getting_started/building_blocks.md (hunk 2 above)
Comment (wensi-ai) [reply docs/getting_started/building_blocks.md:125]: done

File: omnigibson/examples/objects/highlight_objects.py (hunk 6 above)
Comment (wensi-ai) [reply omnigibson/examples/objects/highlight_objects.py:9]: done

File: docs/getting_started/building_blocks.md (hunk 3 above)
Comment (wensi-ai) [reply docs/getting_started/building_blocks.md:368]: done

File: docs/getting_started/building_blocks.md [hunk 7]
```diff
@@ -130,6 +112,25 @@ This demo lets you choose a scene from the BEHAVIOR dataset, loads it along with
...
+
+    * Understanding how to load a scene into **`OmniGibson`**
+    * Accessing all BEHAVIOR dataset scenes
+    * Understanding how to generate a trajectory from a set of waypoints
+
+```{.python .annotate}
+python -m omnigibson.examples.scenes.scene_selector
+```
+
+This demo lets you choose a scene from the BEHAVIOR dataset. It sets the camera to various poses and records images, and then generates a trajectory from a set of waypoints and records the resulting video.
```
Comment (cremebrule) [inline docs/getting_started/building_blocks.md:126]: It allows you to move the camera using the keyboard, select waypoints, and then programmatically generates a video trajectory from the selected waypoints

**Review by cremebrule — APPROVED** [review]
one minor change, then feel free to merge!!

File: docs/getting_started/building_blocks.md (hunk 7 above)
Comment (wensi-ai) [reply docs/getting_started/building_blocks.md]: done

File: docs/getting_started/installation.md (hunk 5 above)
Comment (wensi-ai) [reply docs/getting_started/installation.md]: deleted

**Pull request merged**